test/e2e/syncer/pods/pods.go

```diff
@@ -550,4 +552,88 @@ var _ = ginkgo.Describe("Pods are running in the host cluster", func() {
+			ObjectMeta: metav1.ObjectMeta{
+				Name: podName,
+				Annotations: map[string]string{
+					"vcluster-annotation": "from vCluster with love",
+				},
+				Labels: map[string]string{
+					"vcluster-specific-label": "with_its_value",
+				},
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:            testingContainerName,
+						Image:           testingContainerImage,
+						ImagePullPolicy: corev1.PullIfNotPresent,
+						SecurityContext: f.GetDefaultSecurityContext(),
+					},
+				},
+			},
+		}, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+
+		err = f.WaitForPodRunning(podName, ns)
+		framework.ExpectNoError(err, "A pod created in the vcluster is expected to be in the Running phase eventually.")
+
+		// get current physical Pod resource
+		pPodName := translate.Default.HostName(nil, vPod.Name, vPod.Namespace)
+		pPod, err := f.HostClient.CoreV1().Pods(pPodName.Namespace).Get(f.Context, pPodName.Name, metav1.GetOptions{})
+		framework.ExpectNoError(err)
+
+		// get current vCluster Pod resource
+		vPod, err = f.VClusterClient.CoreV1().Pods(ns).Get(f.Context, podName, metav1.GetOptions{})
+		framework.ExpectNoError(err)
+
+		// update host cluster pod with additional information
+		additionalLabelKey := "another-one"
+		additionalLabelValue := "good-syncer"
+		additionalAnnotationKey := "annotation-key"
+		additionalAnnotationValue := "annotation-value"
+
+		err = wait.PollUntilContextTimeout(f.Context, time.Second, framework.PollTimeout, true, func(context.Context) (bool, error) {
+			if pPod.Labels == nil {
+				pPod.Labels = map[string]string{}
+			}
+			pPod.Labels[additionalLabelKey] = additionalLabelValue
+
+			if pPod.Annotations == nil {
+				pPod.Annotations = map[string]string{}
+			}
+			pPod.Annotations[additionalAnnotationKey] = additionalAnnotationValue
+
+			pPod, err = f.HostClient.CoreV1().Pods(pPod.Namespace).Update(f.Context, pPod, metav1.UpdateOptions{})
+			if err != nil {
+				if kerrors.IsConflict(err) {
+					return false, nil
+				}
+				return false, err
+			}
+
+			return true, nil
+		})
+		framework.ExpectNoError(err)
+
+		// wait for the syncer to update the pod
+		err = wait.PollUntilContextTimeout(f.Context, time.Second, framework.PollTimeout, true, func(ctx context.Context) (bool, error) {
+			pod, err := f.VClusterClient.CoreV1().Pods(vPod.Namespace).Get(ctx, vPod.Name, metav1.GetOptions{})
+			if err != nil {
+				return false, err
+			}
+
+			return pod.ResourceVersion != vPod.ResourceVersion && pod.Annotations[additionalAnnotationKey] != "", nil
+		})
+		framework.ExpectNoError(err)
+
+		vPod, err = f.VClusterClient.CoreV1().Pods(vPod.Namespace).Get(f.Context, vPod.Name, metav1.GetOptions{})
+		framework.ExpectNoError(err)
+
+		framework.ExpectEqual(vPod.Annotations[additionalAnnotationKey], pPod.Annotations[additionalAnnotationKey])
+		framework.ExpectEqual(vPod.Labels[additionalLabelKey], pPod.Labels[additionalLabelKey])
```

Comment:
Should we also test an update on the vPod and check if it propagates to the pPod, similar to the svc test below?!